crates/sui-sdk/src/types.rs

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use sui_json_rpc_types::{
+    BalanceChange, ObjectChange, SuiTransaction, SuiTransactionEffects, SuiTransactionEvents,
+    SuiTransactionResponse,
+};
+use sui_types::digests::TransactionDigest;
+use sui_types::messages_checkpoint::CheckpointSequenceNumber;
+
+#[derive(Debug, Clone)]
+pub struct SuiTransactionFullResponse {
+    pub digest: TransactionDigest,
+    /// Transaction input data
+    pub transaction: SuiTransaction,
+    pub effects: SuiTransactionEffects,
+    pub events: SuiTransactionEvents,
+    pub object_changes: Vec<ObjectChange>,
+    pub balance_changes: Vec<BalanceChange>,
+    pub timestamp_ms: u64,
+    pub confirmed_local_execution: bool,
```

Comment:
This is expected to be None for `get_transactions`

Reply:
can this still be None while running multi_get_transactions_with_options with full_content() after checkpoint is processed?